### PR TITLE
implement-scrollView-in-xml-files

### PR DESCRIPTION
**Pull request by peculiaruc**
Implemeted Scroll view in xml files, added a new releativelayout to achieve this, also added padding to the bottom of the layout

### CODE REVIEW

**Comment by anantcodes** [comment]
> Implemeted Scroll view in xml files, added a new releativelayout to achieve this, also added padding to the bottom of the layout
@peculiaruc 
Please try to test the same on different simulators.
Thank you.

File: app/src/main/res/layout/fragment_competitiveprogramming.xml [hunk 1]
```diff
@@ -1,231 +1,230 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (anantcodes) [inline app/src/main/res/layout/fragment_competitiveprogramming.xml:2]: May not work on all simulators.

**Comment by peculiaruc** [comment]
ok I
 will look into that and adjust it

**Comment by anantcodes** [comment]
> ok I will look into that and adjust it
> […](#)

Yeah sure, thank you!

**Pull request closed without merging**